src/server_manager/css-in-js-rtl-loader.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const postcss = require('postcss');
+const rtl = require('postcss-rtl');
+
+const CSS_PROCESSOR = postcss([rtl()]);
+
+function processCss(css) {
+  return CSS_PROCESSOR.process(css)
+      .css.replace(/\[dir=rtl\]/g, ':host(:dir(rtl))')
```

Comment:
Restore the comments in `generate_rtl_css.js` to explain the replacements.

Reply:
Done